test/verifying_paymaster.test.ts

```diff
@@ -43,9 +43,17 @@ describe('EntryPoint with VerifyingPaymaster', function () {
 
   describe('#parsePaymasterAndData', () => {
     it('should parse data properly', async () => {
-      const paymasterAndData = hexConcat([paymaster.address, defaultAbiCoder.encode(['uint48', 'uint48'], [MOCK_VALID_UNTIL, MOCK_VALID_AFTER]), MOCK_SIG])
+      const paymasterAndData = hexConcat([
```

Comment:
missing helper `packPaymasterAndData(paymasterr, pmGas, postOpGas, data?)`
we don't want to mix "protocol" encoding (pm, gas), with "app" encoding (time-range)